crates/hulk_behavior_simulator/src/game_controller.rs

```diff
@@ -30,12 +33,23 @@ fn game_controller_controller(
         match *command {
             GameControllerCommand::SetGameState(game_state) => {
                 game_controller.state.game_state = game_state;
+                game_controller.state.kicking_team = Team::Opponent;
```

Comment:
Why do you set kicking to to opponent here?

Reply:
Remnant of first test for opponent kickoff, has been remove